app/components/site-chrome/template.hbs

```diff
@@ -148,6 +148,9 @@
     <!-- .x-scrollable-body -->
 
     {{#if media.isLargeAndUp}}
+    <div class="sitechrome__nav-footer">
```

Comment:
why is this `sitechrome` and not `site-chrome`? I'm thinking of how `social-icon-list` is hyphenated.